crates/gpio/src/gpio.rs

```diff
@@ -585,6 +585,11 @@ impl<D: GpioDevice> GpioController<D> {
 
         self.device.set_direction(gpio, dir as u8, value as u32)?;
         state.dir = dir as u8;
+        info!(
```

Comment:
Hmm, why do we want to add such info!() in the global routines? Maybe just add them in the dummy implementation only ?

I also think now, that we probably must rename Dummy to something like MockGpioDevice and move it to a different source file.

Reply:
Moved into the MockGpio impl

Reply:
Moved to the Mock impl